# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tqtinterface/hotfix-installpath.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- tqtinterface-trinity-14.1.4/CMakeLists.txt.vanilla	2025-05-14 20:52:45.398160135 +0200
+++ tqtinterface-trinity-14.1.4/CMakeLists.txt	2025-05-14 20:53:04.726320772 +0200
@@ -51,9 +51,9 @@
 tde_setup_install_path( BIN_INSTALL_DIR     "${EXEC_INSTALL_PREFIX}/bin" )
 
 if( CMAKE_INSTALL_LIBDIR )
-   tde_setup_install_path( LIB_INSTALL_DIR  "${EXEC_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}" )
+   tde_setup_install_path( LIB_INSTALL_DIR  "${CMAKE_INSTALL_LIBDIR}" )
  else()
-   tde_setup_install_path( LIB_INSTALL_DIR  "${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX}" )
+   tde_setup_install_path( LIB_INSTALL_DIR  "lib${LIB_SUFFIX}" )
 endif()
 
 tde_setup_install_path( INCLUDE_INSTALL_DIR   "${CMAKE_INSTALL_PREFIX}/include/tqt" )
